### PR TITLE
Add links to terms of usage and privacy policy

### PR DESCRIPTION
**Pull request by keshan3262**
Screenshot_1623338190

### CODE REVIEW

File: src/components/label/label.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { useLabelStyles } from './label.styles';
 
 interface Props {
   label?: string;
-  description?: string;
+  description?: React.ReactChild | React.ReactChild[];
```
[the remark this answers is not in the document]
Comment (unixvb) [reply src/components/label/label.tsx:9]: and then just
```
{isDefined(description) && <Text style={styles.description}>{description}</Text>}
```

[the remark this answers is not in the document]
Comment (keshan3262) [reply src/components/label/label.tsx:9]: As far as I remember, `View` elements can't be children of `Text` elements

File: src/screens/import-account/import-account.tsx [hunk 2]
```diff
@@ -51,8 +54,21 @@ export const ImportAccount = () => {
...
+                    <Text style={styles.labelText}>the </Text>
+                    <TouchableOpacity onPress={() => openUrl(termsOfUse)}>
+                      <Text style={styles.link}>Terms of Usage</Text>
+                    </TouchableOpacity>
+                    <Text style={styles.labelText}> and </Text>
+                    <TouchableOpacity onPress={() => openUrl(privacyPolicy)}>
+                      <Text style={styles.link}>Privacy Policy</Text>
+                    </TouchableOpacity>
+                  </View>
+                </>
```
Comment (unixvb) [inline src/screens/import-account/import-account.tsx:70]: You are overwriting all `Label` styles.  
Let's contact Vova, design should be updated or `Label` should not be used in this case

File: src/screens/import-account/import-account.tsx [hunk 3]
```diff
@@ -51,8 +54,21 @@ export const ImportAccount = () => {
...
-              description="I have read and agree to
-the Terms of Usage and Privacy Policy"
+              description={
+                <>
+                  <Text style={styles.labelText}>I have read and agree to</Text>
+                  <View style={styles.labelWrapper}>
+                    <Text style={styles.labelText}>the </Text>
+                    <TouchableOpacity onPress={() => openUrl(termsOfUse)}>
+                      <Text style={styles.link}>Terms of Usage</Text>
+                    </TouchableOpacity>
```
Comment (unixvb) [inline src/screens/import-account/import-account.tsx:64]: use `TextLink` component

File: src/components/label/label.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (unixvb) [reply src/components/label/label.tsx:9]: it could)

File: src/screens/import-account/import-account.styles.ts [hunk 4]
```diff
@@ -8,5 +8,21 @@ export const useImportAccountStyles = createUseStyles(({ colors, typography }) =
...
+    flexWrap: 'wrap'
+  },
+  labelText: {
+    ...typography.caption11Regular,
+    color: colors.gray1
+  },
+  link: {
+    ...typography.caption11Semibold,
+    color: colors.blue,
+    textDecorationLine: 'underline'
```
Comment (unixvb) [inline src/screens/import-account/import-account.styles.ts:26]: `labelWrapper` and `link` are not used, remove them

**Pull request merged**